src/i18n/en.json

```diff
@@ -167,7 +167,7 @@
       "op_src_balance_not_found": "Source balance not found",
       "op_reference_duplication": "You cannot make two issuances with the same reference",
       "op_stats_overflow": "Overflow during statistics calculation",
-      "op_limits_exceeded": "Limits exceeded",
+      "op_limits_exceeded": "You have increased the limit. Details in Limits page.",
```

Comment:
`The operation exceeds limits set to your account. Check "Limits" page for your current limits.`